### ISSUE TITLE
Going too fast too suddenly

### ISSUE THREAD

**Issue by nltesown** [skidding/flatris#57]
Great game but it's getting faster too quickly. Couldn't this be made more gradual? It's almost impossible to go beyond 40 rows (in one-player mode).

**Comment by skidding** [comment]
I agree. I want to change this as well. I'll give it a try next week. Can I ask for your help to test?

**Comment by skidding** [comment]
OK, I tweaked some numbers and published the new version at [beta.flatris.space](https://beta.flatris.space). Try it out and let me know what you think!

**Comment by nltesown** [comment]
Fantastic! I'll test it very soon (I play all the time, anyway).

**Comment by skidding** [comment]
@nltesown did you get a chance to try it?

**Comment by nltesown** [comment]
Absolutely. It's definitely more gradulal... but still getting too quickly at the point where I lose control. My average score has improved by 10 lines, I'd say.

**Comment by skidding** [comment]
I'll try to incorporate this [Tetris gravity formula](http://tetris.wikia.com/wiki/Tetris_Worlds#Gravity) when I find some time. Should feel more like the real deal. Although I'm not sure how agressive the original game is with regards to speed, and if it ever stops when it reaches a certain speed or if it keeps speeding up indefinitely.

**Comment by mirrorshades** [comment]
One suggestion might be that as the gravity increases you may want to increase movement left to right as well. Otherwise no one stands a chance to get past 50 lines.
